### DISCUSSION TITLE
Diferenças no Program.cs

### DISCUSSION THREAD

**Discussion by mauricio-snts** [balta-io/forum#49] [Tira Dúvidas] [answered]
Olá!

No módulo 'Fundamentos do ASP.NET 6', criei a aplicação conforme explicado e meu arquivo 'Program.cs' está apresentando informações que não aparecem na aula gravada conforme imagem:

![ImgProgram](https://user-images.githubusercontent.com/56238688/195176322-aff31d84-3298-41aa-afbc-44cb47460cea.jpg)

Meu arquivo .csproj:

![image](https://user-images.githubusercontent.com/56238688/195176583-5e7bb2f4-9838-4eab-97bf-7bcff0210bb4.png)

Porque essa diferença e como ocultar essas informações?

Grato pela ajuda.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @mauricio-snts, como vai?

Estas informações adicionais são colocadas apenas na sua tela, por algum plugin que está usando. No meu caso, que uso o Rider, ele também coloca... ele só está informando o tipo que o `var` será... nada mais que isto.

**Comment by mauricio-snts** [comment]
Olá Baltieri.

Obrigado pela ajuda.

Encontrei a configuração que oculta as informações adicionais.
No arquivo Settings.json removi as linhas marcadas na imagem:



![ImgProgram](https://user-images.githubusercontent.com/56238688/195369474-6a627777-9796-4350-9a68-8d5b68ccd1b2.jpg)


Até onde eu testei, essa alteração não afetou o desenvolvimento da aplicação

**Reply by andrebaltieri** [reply]
Obrigado pelo feedback @mauricio-snts!!!
